src/Tribe/Service_Provider.php

```diff
@@ -50,6 +60,11 @@ protected function hook() {
 
 		tribe( 'tickets.hubspot.api' );
```

Comment:
How many of these classes need to run immediately and not on a different hook? Also, consider that this runs on every page -- are we 100% sure we need all of these on every single page load?

Reply:
I added a todo note, for now during developing having it load every time is useful. 

Once, it is complete I will pair it down to only when required.